_data/ui-text.yml

```diff
@@ -438,6 +438,8 @@ de: &DEFAULT_DE
   comment_success_msg        : "Danke für Ihren Kommentar! Er wird auf der Seite angezeigt, nachdem er geprüft wurde."
   comment_error_msg          : "Entschuldigung, es gab einen Fehler. Bitte füllen Sie alle benötigten Felder aus und versuchen Sie es erneut."
   loading_label              : "Lade..."
+  search_placeholder_text    : "Suchbegriff eingeben..."
+  results_found              : "Erebnis(se) gefunden"
```

Comment:
Actually, there is a typo. it should be `Ergebnis(se) gefunden`

Reply:
You are right. Just fixed it